Lib/dataclasses.py

```diff
@@ -1235,13 +1230,36 @@ def _update_func_cell_for__class__(f, oldcls, newcls):
     # Fix the cell to point to the new class, if it's already pointing
     # at the old class.  I'm not convinced that the "is oldcls" test
     # is needed, but other than performance can't hurt.
-    closure = f.__closure__[idx]
-    if closure.cell_contents is oldcls:
-        closure.cell_contents = newcls
+    cell = f.__closure__[idx]
+    if cell.cell_contents is oldcls:
+        cell.cell_contents = newcls
         return True
     return False
 
 
+def _find_inner_functions(obj, _seen=None, _depth=0):
```

Comment:
Why the leading underscores on `_seen` and `_depth`?

Reply:
This should indicate that these arguments are only used within the function (in recursive calls) and not outside of the function body.

Reply:
I think that the whole function is private, so it can only be used inside CPython by us :)

Reply:
Well, you've got a point. Removed the underscores.